gallery/gallery.py

```diff
@@ -103,9 +103,7 @@ def download_and_extract(package: str, version: str | None, directory: Path) ->
     return directory / result_dir
 
 
-def get_package(
-    package: str, version: str | None, directory: Path
-) -> Path | None:
+def get_package(package: str, version: str | None, directory: Path) -> Path | None:
```

Comment:
```suggestion
def get_package(
    package: str, version: str | None, directory: Path
) -> Path | None:
```

Apparently this file isn't included in CI's formatted check; formatted it in #4867